spec/imsc-hrm.html

```diff
@@ -517,16 +524,18 @@ <h2>General</h2>
         </li>
       </ul>
 
-      <p>The contents of the Presentation Buffer P<sub>n</sub> SHALL be transferred instantaneously to Presentation Buffer
-      P<sub>n-1</sub> at the presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>, making the latter
-      available for display.</p>
+      <p>The contents of the Front Buffer SHALL be transferred instantaneously to the Back Buffer at the presentation
+      time of a <a>non-empty ISD</a> E<sub>n</sub>, making the latter available for display.</p>
+
+      <p>The Back Buffer SHALL be disconnected from the display while <a>empty ISD</a> E<sub>n</sub> is being
+      presented.</p>
```

Comment:
Should we use different subscript letters here, since we're talking about two different ISDs, one that is non-empty and one that is empty? Just thinking it could be confusing.

Reply:
Agree. I think we can simply remove the E_n.

Reply:
Addressed at https://github.com/w3c/imsc-hrm/pull/50/commits/ef20c64433253a25db9b27f72d5bcb881a3d1622